neuraxle/metaopt/auto_ml.py

```diff
@@ -552,7 +580,7 @@ def _fit_data_container(self, data_container: DataContainer, context: ExecutionC
                 self.hyperparams_repository.save_failure_for_trial(hyperparams, error)
 
         if self.refit:
-            self.best_model = self._load_best_model().handle_fit(data_container.copy(), context)
+            self.best_model = self._load_best_model()
```

Comment:
1. A `re-fit` is not simply loading the best model. 
2. A `re-fit` is to take a copy of the "virgin" untouched model, then set on it the best params found (of the actual best model), then fit the virgin (but well-configured) model again on the new incoming data.

Reply:
done in demo